style.css

```diff
@@ -1,82 +1,50 @@
-@import url('https://fonts.googleapis.com/css?family=Muli&display=swap');
-
-* {
-    box-sizing: border-box;
-  }
-  
-  body {
-    font-family: 'Muli', sans-serif;
-    background-color:rgba(87, 87, 240, 0.952);
-    color: blanchedalmond;
-    height: 100vh;
-    overflow: hidden;
-    margin: 0;
-    flex-direction: column;
-    display: flex;
-    align-items: center;
-    height: 100vh;
-    justify-content: center;
-  }
-
-  .container {
-      background-color:white;
-      padding: 40px 80px;
-      border-radius: 10px;
-
-  }
-  .container img {
-      margin-left: 80px;
-      margin-right: auto;
-
-  }
-
-  .container a {
-      text-decoration: none;
-      color: lightblue;
-  }
-
-  .btn {
-     cursor: pointer;
-     display: inline-block;
-     width: 100%;
-     background: rgb(226, 58, 58);
-     padding: 15px;
-     font-family: inherit;
-     font-size: 16px;
-     border: 0;
-     border-radius: 30px;
-  }
-
-  .btn:focus {
-      outline: 0;
-  }
-
-  .btn:active {
-     transform: scale(0.98); 
-  }
-
-  .text {
-      margin-top: 30px;
-  }
-
-  .form-control {
-      position: relative;
-      margin: 20px 0 40px;
-      width: 300px;
-  }
-  .form-control img {
-    margin: 0 -31px -41px;
-  }
-
-  .form-control input {
-    background-color: transparent;
-    border: 0;
-    border-bottom: 2px grey solid;
-    display: block;
-    width: 100%;
-    padding: 15px 0;
-    font-size: 18px;
-    color: #fff;
-  }
-
+body {
+  background-color: #3CA2E0;     
+}
+.container {
+  width: 40%;
+  height: 70%;
+  margin: 100px auto;
+  padding-left: 50px;
+  padding-top: 100px;
+  padding-bottom: 100px;
+  justify-content: center;
+  background-color: white;
+  border-radius: 6px 
+}
+
+input {
+  display: block;
+  border-top: none;
```

Comment:
We can give single property mentioning for all sides, or use two properties

Reply:
corrected